README.md

```diff
@@ -297,6 +297,33 @@ make load-test
 - Review logs that will be sent to Log Analytics when configured
   - See `deploy/loganalytics` for directions
 
+## How Codespaces is built?
```

Comment:
no punctuation in headers

your linter should have flagged this - make sure you have your linter setup